### ISSUE TITLE
Warning "Jest did not exit one second after the test run has completed."

### ISSUE THREAD

**Issue by larstippmann** [SeleniumHQ/selenium-ide#1819]
## 🐛 Bug Report

After running any side file the regardless if successful or not I get an error and it takes from 10 seconds up to a minute until the command has finished.

*****
Jest did not exit one second after the test run has completed.

'This usually means that there are asynchronous operations that weren't stoppedin your tests. Consider running Jest with `--detectOpenHandles` to troubleshootthis issue.
*****

## To Reproduce

npm init
npm i selenium-side-runner@^4.0.5

./node_modules/.bin/selenium-side-runner -c 'browserName=chrome goog:chromeOptions.args=[headless,disable-infobars,disable-gpu,no-sandbox,disable-dev-shm-usage]' -s http://localhost:4444/wd/hub /projects/mpoint/webdriver/test/wuedesto/Wuedesto_DE-Login-Logout.side

I am using Selenium 4.20.0 with Chrome 120.

## Expected behavior

The system should run without error

## Environment

OS: Debian
Selenium IDE Version:  4.2.0
Selenium SIDE Runner Version:  4.0.5
Please note that selenium-side-runner supports Node.js 16+
-->
Node version: 20.12.2
Only required if using selenium-side-runner
-->
Browser: Chrome 120.0.6099.199

**Comment by toddtarsi** [comment]
Thanks for reporting! I'm seeing this as well. Does it hang for a long time on your machine or seem to still cleanup and exit alright?

**Comment by larstippmann** [comment]
It hangs for up to a minute

**Comment by toddtarsi** [comment]
Yeah I'm seeing that too. Not ideal. Will be looking at it this weekend or next. First thing on my slate is doing a bit of changes to the new internationalization system to align that stuff more long term (we are getting some absolutely amazing help from some Chinese developers). After that's out, I'll come back around here.

**Comment by larstippmann** [comment]
Hi Todd, thanks for checking the problem!

**Comment by zjiale** [comment]
I have the same problem in my program TvT

**Comment by eurekatestbed** [comment]
I'm facing a similar issue. Is there a work around for this? 
Thanks in advance.

**Comment by NikhilHedau** [comment]
Hi @toddtarsi 
I am facing this issue in ver 4.0.12 too. Can you please suggest some workaround?
